proto/pocket/migration/tx.proto

```diff
@@ -97,9 +97,12 @@ message MsgClaimMorseAccount {
   // E.g.: 00f9900606fa3d5c9179fc0c8513078a53a2073e
   string morse_src_address = 2 [(gogoproto.jsontag) = "morse_src_address"];
 
+  // The ed25519 public key of the morse account with morse_src_address.
+  bytes morse_public_key = 3 [(gogoproto.casttype) = "crypto/ed25519.PublicKey"];
+
   // The hex-encoded signature, by the Morse account, of this message (where this field is nil).
   // I.e.: morse_signature = private_key.sign(marshal(MsgClaimMorseAccount{morse_signature: nil, ...}))
-  bytes morse_signature = 3 [(gogoproto.jsontag) = "morse_signature"];
+  bytes morse_signature = 4 [(gogoproto.jsontag) = "morse_signature"];
```

Comment:
@okdas is this going to be an issue? We don't have to fix these if so.

Reply:
_tl;dr We ned to do this by the book._

1. There's @red-0ne's alt approach to multiple protobufs
2. This is unlikely an issue because it's not being used
3. All of that said, the risk is not worth it. Please use `reserved` as we're officially in production.